socorro/processor/pipeline.py

```diff
@@ -10,60 +10,13 @@
 """
```

Comment:
I renamed this file from `socorro/processor/processor_pipeline.py` to `socorro/processor/pipeline.py` to take out duplicate information.